src/frontend-samples/missclassification-sample/readme.md

```diff
@@ -0,0 +1,20 @@
+# Misclassifications Sample
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample demonstrates calling Design Validation APIs to retrieve results of a misclassification test and present it for review in a viewer app.
+
+## Purpose
+
+- Populate table of sample misclassification results.
```

Comment:
Maybe mention that you use a DataProvider. This is a simple and practical example of using a DataProvider